src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -498,11 +498,11 @@ bool clang_c_convertert::get_var(const clang::VarDecl &vd, exprt &new_expr)
   symbol.file_local = (vd.getStorageClass() == clang::SC_Static) ||
                       (!vd.isExternallyVisible() && !vd.hasGlobalStorage());
 
-  bool aggregate_value_init = is_aggregate_type(vd.getType());
+  bool aggregate_value = is_aggregate_type(vd.getType());
```

Comment:
Please restore variable name. `dim3 s1 =  {1,2,3};` is copy list initializaiton. For the best software engineering practce, we shouldn't touch any code for aggregate/POD in this patch.